operations/mimir/querier.libsonnet

```diff
@@ -23,6 +23,7 @@
       'querier.frontend-address': if !$._config.is_microservices_deployment_mode || $._config.query_scheduler_enabled then null else
         'query-frontend-discovery.%(namespace)s.svc.%(cluster_domain)s:9095' % $._config,
       'querier.frontend-client.grpc-max-send-msg-size': 100 << 20,
+      'querier.scheduler-client.grpc-max-send-msg-size': 100 << 20,
```

Comment:
Heads-up: We need to be careful here. After we merge the PR, we need to make sure this change doesn't get deployed to all our environments, which don't have the new code yet.